### ISSUE TITLE
Document move of DefaultedOnCreate in upgrade notes

### ISSUE THREAD

**Issue by jsma** [django-salesforce/django-salesforce#275]
I attempted to upgrade to django-salesforce==1.1 recently, but if I try to run ``python manage.py migrate`` I get the following error:

```
AttributeError: module 'salesforce.backend.operations' has no attribute 'DefaultedOnCreate'
```

This meant existing migrations are now broken and it required going through previously applied migrations to update the import and usage:

<img width="2160" alt="Screen Shot 2020-07-31 at 4 39 45 PM" src="https://user-images.githubusercontent.com/24574/89088318-7315b880-d34c-11ea-8b61-f88a5efaea8f.png"> 

It's also not clear, when is ``DEFAULTED_ON_CREATE`` preferred? After running ``inspectdb`` it appears that some fields still use this vs ``DefaultedOnCreate``.

As always, thanks for your work on this! If the problem is between the chair and the computer, apologies in advance but thought I'd raise this issue in case it helps others.

**Comment by hynekcer** [comment]
It can be fixed by adding this line for backward compatibility to `salesforce/backend/operations.py`:

    from salesforce.defaults import DefaultedOnCreate, DEFAULTED_ON_CREATE

or you can upgrade to the current master (8ca907c). It has only 6 different lines of code. (5 lines: difference between v1.1 and v3.1 compatible with Django 3.1 plus the added line above.

or maybe you can delete migrations related to Salesforce models if you save these data only to Salesforce database and a default non-salesforce database is used only for tests or used for some Django models like Django users and permissions.

**Comment by philchristensen** [comment]
@hynekcer I’m glad to do a maintenance release if that would be helpful.

**Comment by jsma** [comment]
Thanks for the update! I had already edited all of the migrations to work, so I'm all set. This appears to fix things for anyone else upgrading (once released), so I'll close this out. Thanks again!

**Comment by hynekcer** [comment]
It is fixed by PR #276 and also by documentation that `salesforce.backend.operations.DefaultedOnCreate`works again, but is deprecated, because the one line fix above had some consequences worse than the original problem.
